packages/volto-dsgvo-banner/src/components/IfConfirm/IfConfirm.jsx

```diff
@@ -36,6 +46,36 @@ const IfConfirm = ({ children, module, showDSGVOBanner }) => {
           </p>
         </Message>
       </div>
+    ) : (
+      <div className="dsgvo-overlay-wrapper">
+        {!isConfirmed && (
+          <div className="dsgvo-overlay">
+            <div className="dsgvo-overlay-content">
+              <p>
+                <FormattedMessage
+                  id="This content is provided by a third-party service. To view it, please allow cookies for {module} in your privacy settings."
+                  defaultMessage="This content is provided by a third-party service. To view it, please allow cookies for {module} in your privacy settings."
+                  values={{
+                    module: <b>{module}</b>,
+                  }}
+                />
+              </p>
```

Comment:
@Tishasoumya-02 This is a better message. Let's use this message when showBanner is true and also when it is false, and only change the button based on the value of showBanner.